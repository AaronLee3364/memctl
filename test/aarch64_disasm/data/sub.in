fffffff0070c0000
d10083ff cb364108 f10400a5 eb08012e
6b2a0108
