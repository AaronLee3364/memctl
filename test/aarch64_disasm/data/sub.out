fffffff0070c0000  SUB     SP, SP, #0x20
fffffff0070c0004  SUB     X8, X8, W22, UXTW
fffffff0070c0008  SUBS    X5, X5, #0x100
fffffff0070c000c  SUBS    X14, X9, X8
fffffff0070c0010  SUBS    W8, W8, W10, UXTB
